lib/handlers/command/slack/AssociateRepo.ts

```diff
@@ -129,6 +128,9 @@ export class AssociateRepo implements HandleCommand {
     @MappedParameter(MappedParameters.SlackChannelName)
     public channelName: string;
 
+    @MappedParameter("atomist://slack/channel_type")
```

Comment:
We probably want that to be a constant in automation-client for consistency. I can do that.

Reply:
Thanks!